src/Microsoft.DotNet.Cli.Utils/Extensions/CollectionsExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public static class CollectionsExtensions
+    {
+        public static IEnumerable<T> EmptyIfNull<T>(this IEnumerable<T> enumerable)
```

Comment:
I like my naming better: OrEmptyIfNull. It reads better when put together with the Enumerable. Like dotnetTest.TestsToRun.OrEmptyIfNull()